### PR TITLE
Brian doyle/zora tutorial

### PR DESCRIPTION
**Pull request by briandoyle81CB**
**What changed? Why?**
Add tutorial using Zora's premint feature

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

**Review by oveddan — APPROVED** [review]
looks great! some minor comments

**Comment by cb-heimdall** [comment]
Review Error for oveddan @ 2024-06-24 15:03:08 UTC
User must have write permissions to review

File: apps/base-docs/tutorials/docs/2_nft-minting-with-zora.md [hunk 1]
```diff
@@ -0,0 +1,442 @@
...
+
+## Getting Started
+
+Begin by making a copy of [An Onchain App in 100 Components] by clicking the `Use this Template` button then cloning it locally.
+
+The team recommends using [Bun], so install it if you need to, then install the packages and run the app:
+
+```bash
+# Install bun in case you don't have it
+bun curl -fsSL <https://bun.sh/install> | bash
```
Comment (hughescoin) [inline apps/base-docs/tutorials/docs/2_nft-minting-with-zora.md:56]: should just be
`curl -fsSL https://bun.sh/install | bash`

File: apps/base-docs/tutorials/docs/2_nft-minting-with-zora.md [hunk 2]
```diff
@@ -0,0 +1,442 @@
...
+
+# Install packages
+bun i
+
+# Run Next app
+bun run dev
+```
+
+You only need to create
```
Comment (hughescoin) [inline apps/base-docs/tutorials/docs/2_nft-minting-with-zora.md:65]: Delete this line?

**Review by hughescoin — APPROVED** [review]

**Pull request merged**